test/resources/restrict-children-sample/localization/csCZ.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
You can leave out the localisation files for other languages than english in the test resources.

Reply:
Just removed them.